api/src/main/java/javax/servlet/ServletContext.java

```diff
@@ -351,12 +351,12 @@
     public RequestDispatcher getNamedDispatcher(String name);
 
     /**
-     * @deprecated As of Java Servlet API 2.1, with no direct replacement.
+     * @deprecated As of Java(tm) Servlet API 2.1, with no direct replacement.
```

Comment:
I have kept the deprecated tags referring to Java Servlet as it was at the time of deprecation, but I have added the (tm) where it was missing.

Reply:
I don't think the (tm) is really necessary...

Reply:
It was on some, but not others, so this was more a consistency thing.  But I'm happy to go with minimal changes now.